StPicoEvent/StPicoTrack.cxx

```diff
@@ -86,3 +96,9 @@ void StPicoTrack::Print(Char_t const* option) const
                                   << nSigmaProton() << "/" << nSigmaElectron()
            << endm;
 }
+
+/***************************************************************************
+ *
+ * $Log$
```

Comment:
I hate it too :-(
But I thought I have to add it since it is in the guidelines. If the reviewers can ignore it I can too ;-)